api/api-doc/storage_service.json

```diff
@@ -917,6 +917,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"ranges_parallelism",
+                     "description":"An integer specifying the number of ranges to repair in parallel.",
```

Comment:
This number is so opaque... What integer should a user use here? 3? 100? 1000000? What is the default if I don't specify this? As a user, I wouldn't know how to use this option (although I guess it might come handy for our field engineers?).

Reply:
The default is used to be 16. Now it is changed to be configured according to the number of memory allowed for repair, see 131acc09cc1d568f994059736c8500adf5d05d88.